packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,389 @@
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch, useSelector } from 'react-redux'
+import { openUrl } from 'src/app/actions'
+import { dappsListApiUrlSelector } from 'src/app/selectors'
+import BackButton from 'src/components/BackButton'
+import BottomSheet from 'src/components/BottomSheet'
+import Dialog from 'src/components/Dialog'
+import CustomHeader from 'src/components/header/CustomHeader'
+import DappsExplorerLogo from 'src/icons/DappsExplorerLogo'
+import LinkArrow from 'src/icons/LinkArrow'
+import Help from 'src/icons/navigator/Help'
+import QuitIcon from 'src/icons/QuitIcon'
+import { TopBarIconButton } from 'src/navigator/TopBarButton'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'DAppExplorerScreen'
+
+const SECTION_HEADER_MARGIN_TOP = 32
+
+interface CategoryWithDapps {
+  id: string
+  name: string
+  fontColor: string
+  backgroundColor: string
+  dapps: Dapp[]
+}
+
+interface Dapp {
+  id: string
+  categoryId: string
+  iconUrl: string
+  name: string
+  description: string
+  dappUrl: string
+}
+
+interface DappProps {
+  dapp: Dapp
+  onPressDapp: (dapp: Dapp) => void
+}
+
+interface SectionData {
+  data: Dapp[]
+  category: CategoryWithDapps
+}
+
+export function DAppsExplorerScreen() {
+  const { t } = useTranslation()
+  const dappsListUrl = useSelector(dappsListApiUrlSelector)
+  const [isHelpDialogVisible, setHelpDialogVisible] = useState(false)
+  const [isBottomSheetVisible, setBottomSheetVisible] = useState(false)
+  const [dappSelected, setDappSelected] = useState<Dapp>()
+  const dispatch = useDispatch()
+
+  const {
+    loading,
+    error,
+    result,
+  }: {
+    loading: boolean
+    error: Error | undefined
+    result: CategoryWithDapps[] | undefined
+  } = useAsync(
+    async () => {
+      if (!dappsListUrl) {
+        throw new Error('Dapps list url is not defined')
+      }
+
+      const response = await fetch(dappsListUrl, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        },
+      })
+
+      const result = await response.json()
+      try {
+        const categoriesById: { [id: string]: CategoryWithDapps } = {}
+        result.categories.forEach((cat: any) => {
+          categoriesById[cat.id] = {
+            id: cat.id,
+            name: cat.name,
+            fontColor: cat.fontColor,
+            backgroundColor: cat.backgroundColor,
+            dapps: [],
+          }
+        })
+        result.applications.forEach((app: any) => {
+          categoriesById[app.categoryId].dapps.push({
+            id: app.id,
+            categoryId: app.categoryId,
+            name: app.name,
+            iconUrl: app.logoUrl,
+            description: app.description,
+            dappUrl: app.url,
+          })
+        })
+
+        return Object.values(categoriesById)
+      } catch (error) {
+        Logger.error(TAG, 'onError', error as Error)
```

Comment:
Maybe add some small detail to the message? Like `Error fetching dApp list` instead of `onError`